<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gruppa</class>
 <widget class="QDialog" name="Gruppa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Группа</string>
  </property>
  <widget class="QTableView" name="groupTableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>701</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="specialityCombo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>351</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="uch_planCombo">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
